.github/workflows/release_win.yml

```diff
@@ -13,42 +14,87 @@ jobs:
       matrix:
         python-version: [3.5, 3.6, 3.7, 3.8]
         architecture: ['x64', 'x86']
-    steps:
-    - uses: actions/checkout@v2
-    - name: Checkout submodules
+    steps:         
+    - name: Checkout Protobuf 
+      uses: actions/checkout@master
+      with:
+        repository: protocolbuffers/protobuf
+        ref: refs/tags/v3.11.3
+        path: ./protobuf_root/protobuf
+        
+    - name: Checkout Protobuf submodules
```

Comment:
You can also set protobuf_BUILD_TESTS to OFF, then this step will not be needed.

Reply:
good point. It is already set to OFF. I can remove this. Closing this PR in favor of the one we checked into rel branch.